.coafile

```diff
@@ -18,4 +18,4 @@ bears = YAMLLintBear
 files = **.yaml, **.yml, .zuul.yaml
 ignore = .zuul.yaml
 max_line_length = 180
-
+yamllint_config = .yamllint
```

Comment:
meh still dont like that :P

Reply:
Why not? I don't think there is a way to set the rules in the coafile directly :frowning_face: